index.bs

```diff
@@ -1509,6 +1513,10 @@ during registration.
         [=attestation statement=], as well as to decode and validate the [=authenticator data=] along with the
         [=JSON-serialized client data=]. For more details, see [[#sctn-attestation]], [[#generating-an-attestation-object]],
         and [Figure 3](#fig-attStructs).
+
+    :   <dfn>transports</dfn>
+    ::  This attribute indicates which [transports](#transport) the [=client=] believes can be used to communicate with
```

Comment:
If the UA gets its belief wrong, what happens? 

For example: A UA using a Feiten U2F token that supports BLE and USB might know that it used USB, but may not know for sure that it has other transports available. It practically mandates decoding `TRANSPORT_EXTENSION_OID`, I think, and maybe should reference it.